tests/all/integration/api/helpers/test_utilities.py

```diff
@@ -33,13 +33,10 @@ def test_string_empty(self):
         """Method to test whether an empty string is correctly identified."""
 
         with app.test_request_context():
-            self.assertTrue(string_empty(''))
-            self.assertTrue(string_empty(' '))
-            self.assertFalse(string_empty('some value'))
-            self.assertFalse(string_empty('  some   value '))
-            self.assertFalse(string_empty(str))
-            self.assertFalse(string_empty(int))
-            self.assertFalse(string_empty(None))
+            self.assertTrue(string_empty('  '))
+            self.assertFalse(string_empty(0))
+            self.assertFalse(string_empty([]))
+            self.assertFalse(string_empty(False))
```

Comment:
Since this is a unit test, move to unit test file please

And open an issue to move all unit tests to the correct directory

Reply:
@iamareebjamal Yeah, I think that should be done right away.